doc/releases/release-notes-3.6.rst

```diff
@@ -1167,17 +1106,15 @@ Libraries / Subsystems
 
```

Comment:
L1104: 
For those devices, Zephyr is able to reduces
->
For those devices, Zephyr is able to reduce
L1105
interruptions -> interrupts